test/test_wofpy_soap.py

```diff
@@ -1,4 +1,4 @@
-import suds
+from suds.client import Client
```

Comment:
This syntax is slightly different from vanilla  `suds` but works on both `suds` and `suds-jurko`. The latter is the Python 2 and 3 compatible version of the former.  (There are some noise due to trailing and blank spaces removal.)